src/client/components/footer/footer.js

```diff
@@ -19,16 +19,22 @@ const Footer = () => {
   return (
     <footer>
       <div className="main-content">
-        <div className="box">
-          <img src={home} alt="home icon" />
-          <h4>HOME</h4>
-        </div>
-        <div className=" box">
-          <Link to={getProfileLink()}>
-            <img src={profile} alt="profile icon" />
-            <h4>PROFILE</h4>
-          </Link>
-        </div>
+        <Router>
```

Comment:
No need to place Router here. Remove it.

Reply:
@tpetrina if i remove <Router>(for Home) from Line22, i get an error as : Invariant failed: You should not use <Link> outside a <Router>. When I use Link, i need Router.

Reply:
Where do you get this error? Is it in storybook? In that case you must use `StaticRouter` _inside_ storybook. Router is already added in `App` and doesn't need to be added twice.

Reply:
Yuka check my PR footer link-router. See how I added the router to storybook file and then add you part on Home in the Footer.js

Reply:
https://github.com/HackYourFuture-CPH/web-dev-rooms/pull/223 solves this issue correctly.

Reply:
@Yaser-Aldammad @tpetrina  thank you so much. Please review my updated PR.